src/scenarios/mauiandroidpodcast/pre.py

```diff
@@ -0,0 +1,45 @@
+'''
+pre-command
+'''
+import requests
+import subprocess
+from mauishared.mauisharedpython import RemoveAABFiles
+from performance.logger import setup_loggers, getLogger
+from shared.precommands import PreCommands
+from shared.versionmanager import versionswritejson, GetVersionFromDllPowershell
+from shared import const
+
+setup_loggers(True)
+precommands = PreCommands()
+target_framework_wo_platform = precommands.framework.split('-')[0]
+
+# Download what we need
+with open ("MauiNuGet.config", "wb") as f:
+    f.write(requests.get(f'https://raw.githubusercontent.com/dotnet/maui/{target_framework_wo_platform}/NuGet.config', allow_redirects=True).content)
```

Comment:
Maybe this could be "maui" helper as well...

Reply:
Combined this with the target_framework_wo_platform line making a single install_versioned_maui method.